### PR TITLE
fixing stories styles part one

### PR DESCRIPTION
**Pull request by mh-alahdadian**
- upgrade storybook and material , fixing dependency conflict
- remove linting rules from tsconfig
- remove overlay scrollbar (you should use the public package instead)
- fix some story styles

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 module.exports = {
   extends: ['react-app'],
-  ignorePatterns: ['dist', 'build', 'node_modules', '.storybook'],
+  ignorePatterns: [
+    'dist',
+    'build',
+    'node_modules',
+    '.storybook',
+    'storybook-static',
+  ],
```
Comment (mdeljavan) [inline .eslintrc.js]: IMO we can use --ignore-path .gitignore instead of ingnorePatterns

File: packages/mui/package.json [hunk 2]
```diff
@@ -61,8 +63,7 @@
     "react-lorem-component": "^0.13.0",
     "react-scripts": "5.0.0",
     "resize-observer-polyfill": "^1.5.1",
-    "stylis-plugin-rtl": "^2.1.1",
-    "tslib": "^2.3.1"
+    "stylis-plugin-rtl": "^2.1.1"
```
Comment (mdeljavan) [inline packages/mui/package.json:66]: I think this package is required, so install either as dependency of peerDependency

File: packages/mui/tsconfig.json [hunk 3]
```diff
@@ -3,6 +3,6 @@
   "include": ["src", "types"],
   "compilerOptions": {
     "rootDir": "./src",
-    "noImplicitReturns": false
+    "jsx": "react-jsx"
```
Comment (mdeljavan) [inline packages/mui/tsconfig.json:6]: tsconfig.base has this option

File: packages/react/tsconfig.json [hunk 4]
```diff
@@ -3,6 +3,6 @@
   "include": ["src", "types"],
   "compilerOptions": {
     "rootDir": "./src",
-    "noImplicitReturns": false
+    "jsx": "react-jsx"
```
Comment (mdeljavan) [inline packages/react/tsconfig.json:6]: remove

File: packages/mui/package.json (hunk 2 above)
Comment (mh-alahdadian) [reply packages/mui/package.json:66]: it only used in stories , package is allowed to use with or without using rtl in theme

File: .eslintrc.js (hunk 1 above)
Comment (mh-alahdadian) [reply .eslintrc.js]: no that way we should write --ignore-path more than once

File: packages/mui/tsconfig.json (hunk 3 above)
Comment (mh-alahdadian) [reply packages/mui/tsconfig.json:6]: done

File: packages/react/tsconfig.json (hunk 4 above)
Comment (mh-alahdadian) [reply packages/react/tsconfig.json:6]: done

**Review by mdeljavan — APPROVED** [review]

**Pull request merged**